content/changelog/2024-01-26-console.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Vercel integration updates, fixes & updates, and more
+---
+
+### New default Postgres version
+
+The default version for newly created Neon projects is now Postgres 16. Neon continues to support Postgres 14 and 15.
+
+### Vercel integration updates
+
+- The environment variables set by the [Neon Vercel Integration](https://vercel.com/integrations/neon) when you add the integration to a Vercel project have changed. Previously, the integration set the following Postgres variables:
+
+    - `DATABASE_URL`
+    - `PGHOST`
+    - `PGUSER`
+    - `PGDATABASE`
+    - `PGPASSWORD`
+
+    The integration now sets **only** the `DATABASE_URL` variable by default, which includes all of the required details for connecting to your Neon database. 
+    
+    If you prefer to use the other Postgres variables with your application instead of `DATABASE_URL`, you can now configure the variables you want on the **Integrations** page in the Neon console. 
+    
+    ![Select Vercel variables](/docs/guides/vercel_select_variables.png)
+
+    The variables you select are set in your Vercel project on your next `git push`. For details, see [Manage Vercel environment variables](/docs/guides/vercel#manage-vercel-environment-variables).
+- Branches created for preview deployments are now created from the primary branch of your Neon project. Previously, the integration created branches from the root branch of your Neon project, which is designated as your project's primary branch by default. Neon allows you to [change your primary branch](/docs/manage/branches#set-a-branch-as-primary). If you have an older version of the integration that creates branches from your root branch and you want branches created from your project's primary branch instead, reinstall the integration from the Vercel Marketplace.
```

Comment:
```suggestion
- Preview deployment branches are now created from the primary branch of your Neon project. Previously, the integration created branches from the root branch of your Neon project, which is designated as your project's primary branch by default. Neon lets you [change your primary branch](/docs/manage/branches#set-a-branch-as-primary). If you have an older version of the integration and you want branches created from your project's primary branch instead of root, reinstall the integration from the Vercel Marketplace.
```

Reply:
Removed this item entirely. It's not available yet. I've noted the wording change for the next changelog.